### PR TITLE
Add pca10059 to the Project & Update Submodules

### PR DESCRIPTION
**Pull request by niksiboxi**
A project for pca10059 a.k.a. USB Dongle

### CODE REVIEW

**Comment by ojousima** [comment]
PCA10059 (dongle) or PCA10056 (devkit)?

File: src/ruuvi.firmware.c.emProject [hunk 1]
```diff
@@ -1059,9 +1305,9 @@
       <file file_name="../nRF5_SDK_15.3.0_59ac345/components/libraries/queue/nrf_queue.c" />
     </folder>
     <folder Name="System">
-      <file file_name="../nRF5_SDK_15.3.0_59ac345/modules/nrfx/mdk/ses_startup_nrf52811.s" />
+      <file file_name="../nRF5_SDK_15.3.0_59ac345/modules/nrfx/mdk/ses_startup_nrf52840.s" />
```
Comment (ojousima) [inline src/ruuvi.firmware.c.emProject:1308]: Were these changes intentional? They seem to be breaking 52811 board

File: src/ruuvi.firmware.c.emProject [hunk 2]
```diff
@@ -1009,8 +1255,8 @@
       linker_printf_fp_enabled="Float"
       linker_printf_width_precision_supported="Yes"
       linker_section_placement_file="$(ProjectDir)/targets/keijo/ses/flash_placement.xml"
-      linker_section_placement_macros="FLASH_PH_START=0x0;FLASH_PH_SIZE=0x30000;RAM_PH_START=0x20000000;RAM_PH_SIZE=0x6000;FLASH_START=0x26000;FLASH_SIZE=0xA000;RAM_START=0x20002d58;RAM_SIZE=0x32A8"
-      linker_section_placements_segments="FLASH RX 0x0 0x80000;RAM RWX 0x20000000 0x10000"
+      linker_section_placement_macros="FLASH_PH_START=0x0;FLASH_PH_SIZE=0x30000;RAM_PH_START=0x20000000;RAM_PH_SIZE=0x6000;FLASH_START=0x26000;FLASH_SIZE=0xA000;RAM_START=0x20002d58;RAM_SIZE=0x3D2A8"
```
Comment (ojousima) [inline src/ruuvi.firmware.c.emProject:1258]: Why the memory size changes on unrelated board?

Comment (niksiboxi) [reply src/ruuvi.firmware.c.emProject]: Due to the large file size, I accidentally changed memory size. Fixed in latest commit.

File: src/ruuvi.firmware.c.emProject (hunk 1 above)
Comment (niksiboxi) [reply src/ruuvi.firmware.c.emProject]: Wrong board indeed. Fixed in latest commit to original parameters.

**Review by niksiboxi — COMMENTED** [review]
Fixed in latest commit.

**Review by ojousima — APPROVED** [review]
Thanks
